contributors/devel/README.md

```diff
@@ -117,10 +117,14 @@ See the [kubernetes/release](https://github.com/kubernetes/release) repository f
 
 ### SIG Scheduling
 
-* **Understanding the Kubernetes Scheduler** [scheduler.md](sig-scheduling/scheduler.md)
+* **Understanding the Kubernetes Scheduler** [scheduling_code_hierarchy_overview.md](sig-scheduling/scheduling_code_hierarchy_overview.md)
 
 * **Scheduler Algorithm in Kubernetes** [scheduler_algorithm.md](sig-scheduling/scheduler_algorithm.md)
 
+* **Understanding how Pods are queued in Kubernetes Scheduler** [scheduler_queues.md](sig-scheduling/scheduler_queues.md)
+
+* **Extending with scheduler framework plugins** [scheduler_framework_plugins.md](sig-scheduling/scheduler_framework_plugins.md)
```

Comment:
Plugins are part of scheduler's extensibility, not all of it. You can remove this bullet for now. We will come up with a dedicated document talking about scheduler's overall extensibility, and then link back here.